micro-rdk/src/esp32/conn/network.rs

```diff
@@ -34,13 +34,10 @@ use esp_idf_svc::{
 use futures_util::lock::Mutex;
 use once_cell::sync::OnceCell;
 
-use crate::{
-    common::credentials_storage::WifiCredentials,
-    esp32::esp_idf_svc::{
-        eth::{BlockingEth, EspEth, OpenEth},
-        sys::EspError,
-    },
-};
+use crate::{common::credentials_storage::WifiCredentials, esp32::esp_idf_svc::sys::EspError};
+
+#[cfg(feature = "qemu")]
```

Comment:
Probably also another ticket/TODO pair: I don't think qemu is going to work on esp32s3 anymore since the OpenEth thing got taken out, and I'm not sure what to do about it.